packages/ui/src/components/organisms/SKUMatrix/styles.scss

```diff
@@ -0,0 +1,162 @@
+[data-fs-sku-matrix-sidebar] {
+  // --------------------------------------------------------
+  // Design Tokens for SKU Matrix Sidebar
+  // --------------------------------------------------------
+
+  // Default properties
+
+  // Background
+  --fs-sku-matrix-sidebar-bkg-color				: var(--fs-color-neutral-0);
+
+  // Title
+  --fs-sku-matrix-sidebar-title-size				: var(--fs-text-size-6);
+  --fs-sku-matrix-sidebar-title-text-weight			: var(--fs-text-weight-semibold);
+
+  // Cell
+  --fs-sku-matrix-sidebar-table-cell-font-size			: var(--fs-text-size-0);
+  --fs-sku-matrix-sidebar-table-cell-text-weight		: var(--fs-text-weight-medium);
+
+  // Font weight
+  --fs-sku-matrix-sidebar-text-weight-medium			: var(--fs-text-weight-medium);
+  --fs-sku-matrix-sidebar-text-weight-semibold			: var(--fs-text-weight-semibold);
```

Comment:
suggestion: I think we don't need these 2 tokens, in line [95](https://github.com/vtex/faststore/pull/2503/files#diff-1ceb862516abbc1b3f1fa241b1da4938c6ee8f2265a80a4f20bd54b2b0f55e77R95) you can just use `var(--fs-sku-matrix-sidebar-table-cell-text-weight)`

Reply:
yeah, make sense !